domain/model/modelmigration/import.go

```diff
@@ -378,3 +360,66 @@ func (i *importModelConstraintsOperation) Execute(
 
 	return nil
 }
+
+type importModelActivatorOperation struct {
```

Comment:
This is great, having an `activator` returned and called inside `importModelOperation` wasn't pretty.